diff --git a/android/build.gradle b/android/build.gradle
index ae141453b78b3f83243ece4fb5f2f63e43379dfe..886a59315e1bd5017c9feb33f468186b712b0db0 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -1,17 +1,7 @@
 import java.nio.file.Paths
 
-buildscript {
-    ext.safeExtGet = {prop ->
-        rootProject.ext.has(prop) ? rootProject.ext.get(prop) : project.properties['ReactNativeWebView_' + prop]
-    }
-    repositories {
-        google()
-        gradlePluginPortal()
-    }
-    dependencies {
-        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${safeExtGet('kotlinVersion')}")
-        classpath("com.android.tools.build:gradle:7.0.4")
-    }
+ext.safeExtGet = {prop ->
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : project.properties['ReactNativeWebView_' + prop]
 }
 
 def getExtOrIntegerDefault(prop) {
